kahuna/public/js/upload/jobs/required-metadata-editor.js

```diff
@@ -44,12 +43,18 @@ jobs.controller('RequiredMetadataEditorCtrl',
     };
 
     ctrl.metadataSearch = (field, q) => {
-
         return mediaApi.metadataSearch(field,  { q }).then(resource => {
             return resource.data.map(d => d.key);
         });
     };
 
+    // As we make a copy of this, we need to watch it
+    // in case the metadata changes from above.
+    $scope.$watch(() => ctrl.originalMetadata, metadata => {
+        console.log(metadata)
```

Comment:
Is it worth moving this higher up alongside the other assignments? Not crucial, just a thought.